### ISSUE TITLE
Expand object?

### ISSUE THREAD

**Issue by wong2** [urish/typewiz#19]
```
function foo(id, opts) {
  console.log({
    id: id,
    platform: opts.platform || '',
  })
}

foo('id', {
  platform: 'web',
})
```

Expect to get:

```
function foo(id: string, opts: {
  platform?: string
}) {
  console.log({
    id: id,
    platform: opts.platform || '',
  })
}

foo('id', {
  platform: 'web',
})
```

**Comment by urish** [comment]
Thanks for checking out the project!

Types for object literals are currently not supported, but it is definitely something I would love to see. A good place to start would be to collect information about the object properties [here](https://github.com/urish/typewiz/blob/4f450afcd7cae8c3ed31a77e0b282927c1f8c84f/src/type-collector-snippet.ts#L11).

**Comment by benjamingr** [comment]
@urish goodness squad!

**Comment by urish** [comment]
Excellent idea @benjamingr!

**Comment by benjamingr** [comment]
@urish and @MadaraUchiha already kindly offered to help :)

**Comment by MadaraUchiha** [comment]
Happy to be voluntold.

**Comment by urish** [comment]
@MadaraUchiha we have a Dev.IL Meetup the same day as the next squad, so I won't be able to attend. But if you want to lead it this time, we can figure out how to make it work

**Comment by benjamingr** [comment]
I say hostile fork!

**Comment by urish** [comment]
Forks for me :)

**Comment by styfle** [comment]
There must be some inside jokes happening here because I don't understand what is happening 🤷‍♂️

**Comment by benjamingr** [comment]
Oh hi @styfle - fancy seeing you :)

We're all friends ""in physical life"", and we're discussing an open-source event that will happen on Mar 21st (in person) (JavaScript Israel Goodness Squad) that sadly @urish won't be able to attend as there is a meetup he's organizing at the same date (Dev.il).

@MadaraUchiha and me talked about this project in StackOverflow chat (we've also worked together at TipRanks) and I volunteered him to write this feature. As Uri won't be able to attend Uri offered that @MadaraUchiha organizes the team working on typewiz during the "hackathon" (which Madara may or may not agree to, although he's totally a loser if he says no).

Sorry for being confusing!
